modin/logging/logger_function.py

```diff
@@ -0,0 +1,28 @@
+from .config import get_logger
+from functools import wraps
+from modin.config import LogMode
+
+
+def logger_decorator(modin_layer: str, function_name: str, log_level: str):
+    def decorator(f):
+        @wraps(f)
+        def run_and_log(*args, **kwargs):
+            if LogMode.get() != "none":
```

Comment:
Even if the check is `== "none"`, I think we still need an `else` clause? But since `return f(*args, **kwargs)` is common code among both cases, I can just remove the `else` and the two lines of code in the current if case that does this. This does not avoid the negative check though.

Reply:
I added a suggested change in another comment to clarify what I meant.